packages/core/android/build.gradle

```diff
@@ -188,10 +188,10 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   compileOnly "com.squareup.okhttp3:okhttp:3.12.13"
   
-  implementation "com.datadoghq:dd-sdk-android-rum:2.6.2"
-  implementation "com.datadoghq:dd-sdk-android-logs:2.6.2"
-  implementation "com.datadoghq:dd-sdk-android-trace:2.6.2"
-  implementation "com.datadoghq:dd-sdk-android-webview:2.6.2"
+  implementation "com.datadoghq:dd-sdk-android-rum:2.10.1"
```

Comment:
Android SDK already has a version 2.11.0 https://github.com/DataDog/dd-sdk-android/releases/tag/2.11.0, it brings mostly tracing code update + Otel + Session Replay updates. Do you want to use this version in this PR instead?

Reply:
You are right. I will give it a try, and if there are no major blockers we can use it.